shell/store/github.js

```diff
@@ -1,239 +1,95 @@
-import dayjs from 'dayjs';
-import { addParam, parseLinkHeader } from '@shell/utils/url';
-import { addObjects, isArray } from '@shell/utils/array';
-import { GITHUB_REPOS, GITHUB_SCOPES, _DATE } from '@shell/config/local-storage';
-
-const API_BASE = 'https://api.github.com/';
-
 export const EXTENDED_SCOPES = ['repo'];
```

Comment:
Now that we're using this store, we should probably ensure that it's cleared on log out. Everything in there at the moment is public, but this may change in the future and it would be good to cover that case now. 
There's a auth store `logout` action where we can call a new `github/reset` action

Reply:
Might not need to do this, see comment on `mutations`